src/filter.c

```diff
@@ -326,7 +326,15 @@ int filter_tee_header (struct filter *chain)
 
 static bool is_blank_line (const char *str)
 {
-	return (regexec (&regex_blank_line, str, 0, NULL, 0) == 0);
+#if 1 /* faster */
+	while (isspace(*str))
+		str++;
+	return (*str == '\0');
+#endif
+#if 0 /* smaller */
```

Comment:
These are just two implementations that you can choose one. Both are faster and take smaller code than the current regex approach.
If you think there is no need to keep two pieces of (functionally equivalent) code, feel free to delete one of them, and have the other into master.